migrations/2022-06-21-123144_language-tags/up.sql

```diff
@@ -0,0 +1,195 @@
+create table language (
+  id serial primary key,
+  code varchar(3),
+  name text
+);
+
+alter table local_user rename column lang to interface_language;
+alter table local_user add column discussion_languages integer[] not null default '{}';
```

Comment:
This is still the main issue: you need to add a `local_user_language` table, to make all the post queries joinable, rather than using `eq_in`)

`(id, local_user_id, language_id)`